aws-iam-role-ci-user-manager/main.tf

```diff
@@ -47,20 +51,6 @@ data "aws_iam_policy_document" "ci-manager" {
 
     resources = ["*"] // list all users to be able to use the console
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single line comments should begin with #